assets/src/editor/events/tickets/editor-ticket/editor-ticket-entities-list.js

```diff
@@ -115,7 +116,7 @@ const createNewTicketEntity = withDispatch(
 			const newTicket = await createEntity( 'ticket', {} );
 			const newBasePrice = await createEntity(
 				'price',
-				{ prtId: basePriceTypeEntity.id }
+				{ PRT_ID: basePriceTypeEntity.id }
```

Comment:
When creating new entities, the _actual_ field names must be used (not the aliases).